src/servers/src/prom.rs

```diff
@@ -463,6 +478,191 @@ pub async fn range_query(
     PromJsonResponse::from_query_result(result, metric_name, Some(ValueType::Matrix)).await
 }
 
+#[derive(Debug, Default, Serialize, JsonSchema)]
+struct Matches(Vec<String>);
+
+#[derive(Debug, Default, Serialize, Deserialize, JsonSchema)]
+pub struct LabelsQuery {
+    start: Option<String>,
+    end: Option<String>,
+    #[serde(flatten)]
+    matches: Matches,
```

Comment:
I think we can use this?
```rust
#[serde(rename = "match[]")]
matches: Option<Vec<String>>
```

Reply:
`match[]` seems to be a (array of) valid promql query. 

>maybe it also can't deserialize duplicate match[]

We don't need to handle duplicate `match[]`